app/src/ui/add-repository/create-repository.tsx

```diff
@@ -81,11 +83,25 @@ export class CreateRepository extends React.Component<ICreateRepositoryProps, IC
 
     const licenses = await getLicenses()
     this.setState({ ...this.state, licenses })
+
+    try {
+      await this.ensureDirectory(this.state.path)
+    }
+    catch (e) {
+      //We don't need the error here
+    }
   }
 
-  private onPathChanged = (event: React.FormEvent<HTMLInputElement>) => {
-    const path = event.currentTarget.value
-    this.setState({ ...this.state, path })
+  private onPathChanged = async (event: React.FormEvent<HTMLInputElement>) => {
+    try {
+      const path = event.currentTarget.value
+
+      this.setState({ ...this.state, path })
+      await this.ensureDirectory(path)
```

Comment:
Just a heads up: `FSE.ensureDir` will probably fire for each keystroke here, and as per [the docs](https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir.md) you might create a lot of unnecessary folders.

> Ensures that the directory exists. **If the directory structure does not exist, it is created. **